src/WebJobs.Script.WebHost/DependencyInjection/DryIoc/Container.cs

```diff
@@ -6574,7 +6580,8 @@ protected virtual Expr ApplyReuse(Expr serviceExpr, Request request)
                 !request.TracksTransientDisposable &&
                 !request.IsWrappedInFunc())
             {
-                var factoryDelegate = serviceExpr.CompileToFactoryDelegate();
+                var container = request.Container as Container;
```

Comment:
Do we need check for "null" here?

Reply:
it will never be null but will add the check just to be sure 100%

Reply:
actually it will never been null.